src/app/barter/effects/transactions.effects.ts

```diff
@@ -32,6 +32,10 @@ export class TransactionsEffects {
     )
   );
 
+  observeTransactions$ = createEffect(() => this.transactionsService.getAll$().pipe(skip(1),
```

Comment:
why skip?

Reply:
For skipping the first time it's happening, when the app is up for the first time